### ISSUE TITLE
Don't output yaml file content during `box add`

### ISSUE THREAD

**Issue by patcon** [spil-ruslan/vagrant-box-updater#3]
```
➜  proviso git:(8-puppet-support) vagrant box add precise64 --force http://files.vagrantup.com/precise64.box
Downloading or copying the box...
Extracting box...te: 752k/s, Estimated time remaining: --:--:--)
Successfully added box 'precise64' with provider 'virtualbox'!
Box details: ---
modification_date: Fri, 14 Sep 2012 06:31:33 GMT
url: http://files.vagrantup.com/precise64.box
➜  proviso git:(master)  
```

Thinking that this level of output might fit best in debug output. Perhaps we could just say (in proper vagrant output log formatting):

```
Box details saved for tracking remote box updates.
```

Or something...? :)

**Comment by spil-ruslan** [comment]
Hi yes you're right. I've reduced verbosity. Should be available through gem as version 0.0.3

**Comment by patcon** [comment]
Thanks @spil-ruslan! I'll try to contribute more fixes like this (admittedly very minor ones) once I start using this is a project that I'm working on!
